.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+CHANGELOG: `no-impact` / `Added` / `Changed` / `Deprecated` / `Removed` / `Fixed` / `Security`
+
+Describe how these changes affect the customer in one line. This is the thing you are going to add to a changelog.txt file.
+
+> CHANGELOG entries will be pulled directly into the release changelog, so use
+customer-facing language. Start with one of the active verbs above,
+write a full sentence, and end with a period. If these changes do not
+impact the customer, then this line should read CHANGELOG: no-impact.
+
+### Description
+Describe why you're making this change, in plain English.
+
+### Testing & Repro steps
```

Comment:
Tiny nit, but I think we should avoid using abbreviations that new folks might not be familiar with.

```suggestion
### Testing & Reproduction steps
```